low-level/sensors/impl/platforms/realstor/realstor_enclosure_sensor.py

```diff
@@ -131,7 +150,8 @@ def run(self):
 
                 self.fault_alert = True
 
-            elif mc_timeout_counter == 0 and self.previous_alert_type != self.rssencl.FRU_FAULT_RESOLVED \
+            elif mc_timeout_counter == 0 and  ws_response_status == 200 \
```

Comment:
what is this logic change, why necessary? Why we are touching unrelated code?

@Sumedh-a-k  in general, dont increase scope of the change beyond whats discussed, it increases test scope too.

Reply:
sure, as explained in the previous comment, due to persistent cache addition, we need this variable.